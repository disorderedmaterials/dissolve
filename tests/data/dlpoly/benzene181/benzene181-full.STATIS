Disorder Model                                                                  
 ENERGY UNITS=kjoule/mol 
         1  5.000000E-10        38
  1.319694E+04  2.980000E+02  5.128231E+03 -1.334653E+03  5.612389E+02
  2.354203E+03  1.685309E+03  1.862134E+03  0.000000E+00  1.769047E+04
  0.000000E+00  2.656844E+03 -5.245340E+04 -9.332635E+02  5.604351E+04
  0.000000E+00  0.000000E+00  0.000000E+00  2.679825E+04  0.000000E+00
  0.000000E+00  0.000000E+00  9.000000E+01  9.000000E+01  9.000000E+01
  0.000000E+00  2.747980E+00  1.574839E-17  1.808866E-16  2.824376E+00
  7.044601E-01 -1.980550E-01  7.044601E-01  1.891433E+00  9.367146E-01
 -1.980550E-01  9.367146E-01  3.528131E+00
