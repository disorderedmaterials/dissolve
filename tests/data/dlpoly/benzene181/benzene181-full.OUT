
                    DL_POLY Classic 1.9

                              Running on    1 nodes


 ************************************************************************************************************************
 ************************************************************************************************************************
 ************************************************************************************************************************
 ***************     DL_POLY CONTROL file                                                                 ***************
 ************************************************************************************************************************
 ************************************************************************************************************************
 ************************************************************************************************************************


 SIMULATION CONTROL PARAMETERS


 simulation temperature           2.9800E+02

 Nose-Hoover 
 thermostat relaxation time       1.0000E-01

 simulation timestep              5.0000E-10

 selected number of timesteps            1

 real space cut off               1.2000E+01

 real space cut off (vdw)         1.2000E+01

 border width of Verlet shell     5.0000E-01


 Electrostatics : Shifted Coulombic potential

 trajectory file option on  
 trajectory file start                   1
 trajectory file interval             1000
 trajectory file info key                0

 data printing interval                  1

 data stacking interval                  1

 statistics file interval                1

 user allocated job time (d)    ********

 job closure time        (s)    1000.000


SYSTEM SPECIFICATION

 energy units=kjoule/mol 


 number of molecular types               1

 molecular species type                  1


 name of species:             C6H6                                    

 number of molecules                   181

 number of atoms/sites                  12


 atomic characteristics:

                      site     name          mass        charge    repeat    freeze

                         1     CA          12.01100    -0.11500         1         0
                         2     HA           1.00800     0.11500         1         0
                         3     CA          12.01100    -0.11500         1         0
                         4     HA           1.00800     0.11500         1         0
                         5     CA          12.01100    -0.11500         1         0
                         6     HA           1.00800     0.11500         1         0
                         7     CA          12.01100    -0.11500         1         0
                         8     HA           1.00800     0.11500         1         0
                         9     CA          12.01100    -0.11500         1         0
                        10     HA           1.00800     0.11500         1         0
                        11     CA          12.01100    -0.11500         1         0
                        12     HA           1.00800     0.11500         1         0

 number of chemical bonds               12


 chemical bond details:

                            key     index     index                            parameters

                           harm         1         2     3.071056E+03   1.080000E+00   0.000000E+00   0.000000E+00
                           harm         1         3     3.924592E+03   1.400000E+00   0.000000E+00   0.000000E+00
                           harm         1        11     3.924592E+03   1.400000E+00   0.000000E+00   0.000000E+00
                           harm         3         4     3.071056E+03   1.080000E+00   0.000000E+00   0.000000E+00
                           harm         3         5     3.924592E+03   1.400000E+00   0.000000E+00   0.000000E+00
                           harm         5         6     3.071056E+03   1.080000E+00   0.000000E+00   0.000000E+00
                           harm         5         7     3.924592E+03   1.400000E+00   0.000000E+00   0.000000E+00
                           harm         7         8     3.071056E+03   1.080000E+00   0.000000E+00   0.000000E+00
                           harm         7         9     3.924592E+03   1.400000E+00   0.000000E+00   0.000000E+00
                           harm         9        10     3.071056E+03   1.080000E+00   0.000000E+00   0.000000E+00
                           harm         9        11     3.924592E+03   1.400000E+00   0.000000E+00   0.000000E+00
                           harm        11        12     3.071056E+03   1.080000E+00   0.000000E+00   0.000000E+00

 number of bond angles                  18


 bond angle details:

                            key     index     index     index     f-const       angle

                           harm         2         1         3  2.9288E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         2         1        11  2.9288E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         3         1        11  5.2718E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         1         3         4  2.9288E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         1         3         5  5.2718E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         4         3         5  2.9288E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         3         5         6  2.9288E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         3         5         7  5.2718E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         6         5         7  2.9288E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         5         7         8  2.9288E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         5         7         9  5.2718E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         8         7         9  2.9288E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         7         9        10  2.9288E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         7         9        11  5.2718E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm        10         9        11  2.9288E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         1        11         9  5.2718E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         1        11        12  2.9288E+02  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         9        11        12  2.9288E+02  120.000000    0.000000    0.000000    0.000000    0.000000

 number of dihedral angles              24


 dihedral angle details:

                            key     index     index     index     index     f-const       angle        trig    1-4 elec     1-4 vdw

                           cos3         2         1         3         4  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         2         1         3         5  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3        11         1         3         4  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3        11         1         3         5  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         2         1        11         9  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         2         1        11        12  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         3         1        11         9  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         3         1        11        12  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         1         3         5         6  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         1         3         5         7  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         4         3         5         6  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         4         3         5         7  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         3         5         7         8  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         3         5         7         9  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         6         5         7         8  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         6         5         7         9  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         5         7         9        10  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         5         7         9        11  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         8         7         9        10  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         8         7         9        11  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         7         9        11         1  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3         7         9        11        12  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3        10         9        11         1  0.0000E+00   30.334000    0.000000    0.500000    0.500000
                           cos3        10         9        11        12  0.0000E+00   30.334000    0.000000    0.500000    0.500000


 number of specified pair potentials         3


                atom 1  atom 2      key                              parameters


                CA      CA        lj       2.92880E-01  3.55000E+00
                CA      HA        lj       1.91735E-01  2.93104E+00
                HA      HA        lj       1.25520E-01  2.42000E+00

 configuration file name: 

          Disorder Model                                                                  


 selected image convention               1


 simulation cell vectors


                        29.925090    0.000000    0.000000
                         0.000000   29.925090    0.000000
                         0.000000    0.000000   29.925090


 system volume            26798.2474300    

  *** warning - : 1..4 scale factors reset for mole    1     sites          3         9

  *** warning - : 1..4 scale factors reset for mole    1     sites          5        11

  *** warning - : 1..4 scale factors reset for mole    1     sites          7         1


 total degrees of freedom                      6513.
 rotational degrees of freedom                    0.
 shell pseudo degrees of freedom                  0.


 sample of starting configuration

      i       x(i)        y(i)        z(i)       vx(i)       vy(i)       vz(i)       fx(i)       fy(i)       fz(i)


      1  9.9443E+00  1.3622E+01 -6.4585E+00  9.4235E+00 -2.3627E+00 -5.0432E+00 -1.5218E+03  5.5780E+03 -6.3348E+03
    110 -1.3173E+01  5.4372E+00 -1.1747E+01 -1.3262E+01 -6.4179E+00  2.1034E+00  3.7415E+03  1.7890E+03 -1.1581E+01
    219 -4.2206E+00 -1.3812E+01 -8.5995E+00 -1.5205E+00 -1.2740E+00  2.4537E+00  7.1828E+03 -2.3089E+04 -1.7624E+03
    328 -5.5143E+00 -7.6332E+00 -8.5263E+00  1.6455E+01  5.2018E+00 -1.2170E+01  4.9785E+03 -4.9283E+03 -7.5575E+03
    437 -1.0963E+01  8.4281E+00  1.2696E+00  6.0151E+00  2.1182E+00  3.9692E-01 -7.3572E+03 -4.1674E+03 -1.5245E+04
    546 -4.4830E+00  1.4575E+01 -1.1030E+01  1.3017E+00 -1.4475E+00  1.7590E+01 -4.0755E+03  5.7037E+02 -3.0360E+03
    655  1.0773E+01  1.4835E+01 -1.1436E+01 -2.9499E+00  1.0003E+00 -1.1543E+01 -1.4157E+03  2.5729E+04  4.0148E+03
    764  9.6974E+00  3.5020E+00 -1.1305E+00 -1.0268E+01 -1.5540E+01 -1.9245E+01 -1.0432E+03  1.6158E+04  9.9030E+02
    873  8.5664E+00 -6.2466E+00  1.1498E+01  5.5452E+00 -5.8281E+00 -2.7761E+00 -5.5342E+03  2.6549E+04  1.0975E+04
    982 -1.9235E+00  5.7123E+00  1.1689E+01 -2.1391E+01 -2.8086E+01 -1.1908E+01 -4.6416E+03  1.3837E+03 -6.6668E+03
   1091  9.7518E+00  7.6085E-01  9.7952E+00 -2.4584E+00  7.5289E+00 -1.3171E+00 -9.8531E+03  4.1077E+03  8.3867E+03
   1200  1.1684E+01 -2.0885E+00 -1.4651E+01 -2.3846E+00  8.4738E+00 -2.1337E+01 -3.4327E+03 -6.8154E+02 -3.9996E+03
   1309 -5.1440E+00 -7.9673E+00 -4.2994E+00  2.9018E+00  2.8447E+00 -9.0888E+00 -5.2714E+03  4.4050E+03 -1.3518E+04
   1418  2.7479E+00  6.3596E+00  1.1444E+01  1.4201E+01  1.1704E+01 -3.3079E+01 -1.2902E+03  9.3438E+03  6.0798E+03
   1527 -5.8097E+00 -1.3087E+01 -1.3713E+01  5.7737E+00  9.8484E-01 -3.9022E+00 -9.6497E+03 -3.7002E+02  2.5935E+04
   1636  1.4163E+01  8.3144E+00 -6.3667E+00  1.3726E+01 -4.0142E+00 -1.6612E+00  6.5661E+03  3.5045E+03 -1.8588E+03
   1745  8.7011E+00  9.5572E+00  2.8305E+00 -6.7994E+00  8.0011E-01  4.9600E+00 -1.1563E+04  7.1693E+03 -1.0979E+03
   1854  1.1247E+01 -1.2575E+01 -4.6799E+00 -1.5536E+01 -3.8328E+01  1.1576E+01 -2.3030E+02  2.6236E+03  4.6026E+02
   1963 -1.4314E+01 -2.9481E+00 -4.8159E+00 -3.9413E+00 -1.5133E+00  1.7128E-01  1.4385E+04  4.2522E+03 -9.2843E+03
   2072  1.1871E+01  9.1332E+00  1.4351E+01 -3.2961E+01 -1.5789E+01 -1.3907E+00  3.7356E+03 -8.6808E+02 -3.2429E+03


long range correction for: vdw energy    -0.182307E+03
                         : vdw pressure  -0.222938E+00

 time elapsed since job start =           0.014 seconds

 ------------------------------------------------------------------------------------------------------------------------

     step     eng_tot    temp_tot     eng_cfg     eng_vdw     eng_cou     eng_bnd     eng_ang     eng_dih     eng_tet
 time          eng_pv    temp_rot     vir_cfg     vir_vdw     vir_cou     vir_bnd     vir_ang     vir_con     vir_tet
 cpu time      volume    temp_shl     eng_shl     vir_shl       alpha        beta       gamma     vir_pmf       press

 ------------------------------------------------------------------------------------------------------------------------
        1  1.3197E+04  2.9800E+02  5.1282E+03 -1.3347E+03  5.6124E+02  2.3542E+03  1.6853E+03  1.8621E+03  0.0000E+00
   0.000f  1.7690E+04  0.0000E+00  2.6568E+03 -5.2453E+04 -9.3326E+02  5.6044E+04  0.0000E+00  0.0000E+00  0.0000E+00
   0.095s  2.6798E+04  0.0000E+00  0.0000E+00  0.0000E+00  9.0000E+01  9.0000E+01  9.0000E+01  0.0000E+00  2.7480E+00

  rolling  1.3197E+04  2.9800E+02  5.1282E+03 -1.3347E+03  5.6124E+02  2.3542E+03  1.6853E+03  1.8621E+03  0.0000E+00
 averages  1.7690E+04  0.0000E+00  2.6568E+03 -5.2453E+04 -9.3326E+02  5.6044E+04  0.0000E+00  0.0000E+00  0.0000E+00
           2.6798E+04  0.0000E+00  0.0000E+00  0.0000E+00  9.0000E+01  9.0000E+01  9.0000E+01  0.0000E+00  2.7480E+00
 ------------------------------------------------------------------------------------------------------------------------


 run terminating. elapsed cpu time =   1.01000E-01, job time = *******d, close time = 1000.00s



 run terminated after       1 steps. final averages calculated over       1 steps.


 ------------------------------------------------------------------------------------------------------------------------

     step     eng_tot    temp_tot     eng_cfg     eng_vdw     eng_cou     eng_bnd     eng_ang     eng_dih     eng_tet
 time          eng_pv    temp_rot     vir_cfg     vir_vdw     vir_cou     vir_bnd     vir_ang     vir_con     vir_tet
 cpu time      volume    temp_shl     eng_shl     vir_shl       alpha        beta       gamma     vir_pmf       press

 ------------------------------------------------------------------------------------------------------------------------
        1  1.3197E+04  2.9800E+02  5.1282E+03 -1.3347E+03  5.6124E+02  2.3542E+03  1.6853E+03  1.8621E+03  0.0000E+00
   0.000f  1.7690E+04  0.0000E+00  2.6568E+03 -5.2453E+04 -9.3326E+02  5.6044E+04  0.0000E+00  0.0000E+00  0.0000E+00
   0.116s  2.6798E+04  0.0000E+00  0.0000E+00  0.0000E+00  9.0000E+01  9.0000E+01  9.0000E+01  0.0000E+00  2.7480E+00

  r.m.s.   0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00
 fluctn.   0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00
           0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00
 ------------------------------------------------------------------------------------------------------------------------


            Approximate 3D Diffusion  coefficients (10^-9 m^2 / s)

            atom        D 
            CA         0.0000E+00
            HA         0.0000E+00


                Average pressure tensor                                       r.m.s. fluctuations 

           2.8244E+00  7.0446E-01 -1.9806E-01                          0.0000E+00  0.0000E+00  0.0000E+00
           7.0446E-01  1.8914E+00  9.3671E-01                          0.0000E+00  0.0000E+00  0.0000E+00
          -1.9806E-01  9.3671E-01  3.5281E+00                          0.0000E+00  0.0000E+00  0.0000E+00

            trace/3.   2.7480E+00


 sample of final configuration

      i       x(i)        y(i)        z(i)       vx(i)       vy(i)       vz(i)       fx(i)       fy(i)       fz(i)


      1  9.9443E+00  1.3622E+01 -6.4585E+00  9.4235E+00 -2.3627E+00 -5.0432E+00 -1.6108E+03  5.6325E+03 -6.5704E+03
    110 -1.3173E+01  5.4372E+00 -1.1747E+01 -1.3262E+01 -6.4179E+00  2.1034E+00  3.5535E+03  1.6009E+03 -4.1373E+02
    219 -4.2206E+00 -1.3812E+01 -8.5995E+00 -1.5205E+00 -1.2740E+00  2.4537E+00  7.0720E+03 -2.3185E+04 -1.7608E+03
    328 -5.5143E+00 -7.6332E+00 -8.5263E+00  1.6455E+01  5.2018E+00 -1.2170E+01  4.9871E+03 -5.0311E+03 -7.6643E+03
    437 -1.0963E+01  8.4281E+00  1.2696E+00  6.0151E+00  2.1182E+00  3.9692E-01 -7.0967E+03 -4.3150E+03 -1.5259E+04
    546 -4.4830E+00  1.4575E+01 -1.1030E+01  1.3017E+00 -1.4475E+00  1.7590E+01 -4.1108E+03  6.8131E+02 -2.9943E+03
    655  1.0773E+01  1.4835E+01 -1.1436E+01 -2.9499E+00  1.0003E+00 -1.1543E+01 -1.4392E+03  2.5923E+04  4.1504E+03
    764  9.6974E+00  3.5020E+00 -1.1305E+00 -1.0268E+01 -1.5540E+01 -1.9245E+01 -9.7992E+02  1.5932E+04  9.6199E+02
    873  8.5664E+00 -6.2466E+00  1.1498E+01  5.5452E+00 -5.8281E+00 -2.7761E+00 -5.4070E+03  2.6179E+04  1.1229E+04
    982 -1.9235E+00  5.7123E+00  1.1689E+01 -2.1391E+01 -2.8086E+01 -1.1908E+01 -4.7142E+03  1.4691E+03 -6.8654E+03
   1091  9.7518E+00  7.6085E-01  9.7952E+00 -2.4584E+00  7.5289E+00 -1.3171E+00 -9.9698E+03  3.9707E+03  8.4317E+03
   1200  1.1684E+01 -2.0885E+00 -1.4651E+01 -2.3846E+00  8.4738E+00 -2.1337E+01 -3.3861E+03 -6.5004E+02 -3.9386E+03
   1309 -5.1440E+00 -7.9673E+00 -4.2994E+00  2.9018E+00  2.8447E+00 -9.0888E+00 -5.5082E+03  4.2551E+03 -1.3569E+04
   1418  2.7479E+00  6.3596E+00  1.1444E+01  1.4201E+01  1.1704E+01 -3.3079E+01 -1.3171E+03  9.4403E+03  6.1473E+03
   1527 -5.8097E+00 -1.3087E+01 -1.3713E+01  5.7737E+00  9.8484E-01 -3.9022E+00 -9.7052E+03 -3.9736E+02  2.5816E+04
   1636  1.4163E+01  8.3144E+00 -6.3667E+00  1.3726E+01 -4.0142E+00 -1.6612E+00  6.6816E+03  3.6298E+03 -1.7749E+03
   1745  8.7011E+00  9.5572E+00  2.8305E+00 -6.7994E+00  8.0011E-01  4.9600E+00 -1.1726E+04  6.9909E+03 -1.3536E+03
   1854  1.1247E+01 -1.2575E+01 -4.6799E+00 -1.5536E+01 -3.8328E+01  1.1576E+01 -2.1272E+02  2.7133E+03  4.8629E+02
   1963 -1.4314E+01 -2.9481E+00 -4.8159E+00 -3.9413E+00 -1.5133E+00  1.7128E-01  1.4378E+04  4.2654E+03 -9.4023E+03
   2072  1.1871E+01  9.1332E+00  1.4351E+01 -3.2961E+01 -1.5789E+01 -1.3907E+00  3.6932E+03 -8.6945E+02 -3.2331E+03

 time elapsed since job start =           0.116 seconds

